proto/sentry_protos/snuba/v1/endpoint_aggregate_bucket.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/request_common.proto";
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+import "sentry_protos/snuba/v1/trace_item_filter.proto";
+
+enum Function {
```

Comment:
Are the `min`, `max`, `p90` and `uniq` [left out](https://www.notion.so/sentry/Qu-EAP-Querying-Events-Analytics-Platform-76629acc212b4ac486c63e7979f16076?pvs=4#817970de59f7455093457621a621dd19) on purpose?